initfiles/bin/init_dfuserver

```diff
@@ -25,9 +25,9 @@ echo $$ > $INIT_PID_NAME
 export SENTINEL="dfuserver.sentinel"
 rm -f ${SENTINEL}
 
-ulimit -c unlimited
-ulimit -n 8192
-ulimit -s 512
```

Comment:
Dont see anywhere stack size is adjusted in the new method ?
Is 512 meant to lower it or raise it from the default ?